addons/hr_work_entry_holidays/__manifest__.py

```diff
@@ -14,7 +14,7 @@
 
 This application allows you to integrate time off in payslips.
     """,
-    'depends': ['hr_holidays', 'hr_work_entry_contract'],
+    'depends': ['hr_holidays', 'hr_holidays_contract', 'hr_work_entry_contract'],
```

Comment:
My 2 cents: better to be explicit. Can `hr_work_entry_holidays` fully work if `hr_holidays_contract` is uninstalled (after being auto-installed for example)? If yes, then no need to add it here; if not, then it _must_ be here. Note that implicit auto-install dependencies, that are actually hard-dependencies, would fail sooner than later in upgrades (as soon as a client uninstalled the auto-install module). Unfortunately runbot won't test these cases.

Reply:
Makes sense. I guess it's the `hr_holidays` dependency that should have been removed in that case, to avoid redundancy.

Reply:
Indeed.